5a/SiloHost/src/Program.fs

```diff
@@ -116,8 +124,35 @@ let main args =
     printfn $"Gateway Port: {gatewayPort}"
     printfn $"Primary Silo Port: {primarySiloPort}"
     printfn $"Dashboard Port: {dashboardPort}"
-
-    //TODO Add Orleans host
-
+    
+    let configureLogging (builder : ILoggingBuilder) =
```

Comment:
This works for now, but I guess we can make it prettier...

Reply:
Ah it's fine 👍